systemverilog-plugin/UhdmAst.cc

```diff
@@ -722,8 +729,7 @@ static void convert_packed_unpacked_range(AST::AstNode *wire_node)
     if (convert_node) {
         // if not already converted
         if (wire_node->multirange_dimensions.empty()) {
-            const size_t packed_size = add_multirange_attribute(wire_node, packed_ranges);
-            const size_t unpacked_size = add_multirange_attribute(wire_node, unpacked_ranges);
+            const auto [packed_size, unpacked_size] = set_multirange_dimensions(wire_node, packed_ranges, unpacked_ranges);
```

Comment:
Could you please check, if changing above ``if`` to assert is possible?
```
log_assert(wire_node->multirange_dimensions.empty());
```
I think we shouldn't convert single range multiple times, but I'm not sure if this condition is upheld in current code.

Reply:
I've checked it, and the tests fail. At the moment we're not able to replace the if with an assert.